mantis-control-plane/mantis-control-plane-core/src/main/java/io/mantisrx/server/core/master/LocalMasterMonitor.java

```diff
@@ -41,4 +41,19 @@ public Observable<MasterDescription> getMasterObservable() {
     public MasterDescription getLatestMaster() {
         return master;
     }
+
+    @Override
+    public void start() {
```

Comment:
This is another potential controversial change. The master monitor was a service like class because the curator was started outside of it, and the curator was a service. When I made the choice to reduce the exposure of the curator to zookeeper the need to start/start (service manage) the monitor needed to move as well. It made sense to me for this to be a service.

However, I can't make it a `BaseService` without making an abstract class. I stopped at implementing the service interface, but not a strongly held position.